src/recommendations/RecommendationsPage.jsx

```diff
@@ -24,10 +24,15 @@ const RecommendationsPage = (props) => {
 
   const { formatMessage } = useIntl();
   const [isLoading, setIsLoading] = useState(true);
+  const [selectedOption, setSelectedOption] = useState(RECOMMENDATIONS_OPTION_LIST[0]);
```

Comment:
This state should live in `RecommendationsList` component instead of having it here and passing as prop.

Reply:
as per slack correspondance we are ignoring it